python.md

```diff
@@ -0,0 +1,64 @@
+Python Coding Standards
+=======================
+
+Overview
+--------
+This section contains the source documentation for our python guidelines but is ultimately a lot of reading. Please see the Next Section for easy setup instructions that will adhere to the following principles.
+Python code within our repositories should adhere to the [Quality Assurance of Code for Analysis and Research](https://best-practice-and-impact.github.io/qa-of-code-guidance/index.html) created by the Office for National Statistics. This naturally builds on top of the [PEP 8](https://www.python.org/dev/peps/pep-0008/) globally accepted python formatting guide as well as emphasising the [Reproducible Analytical Pipelines Minimum Viable Product Guidance](https://github.com/best-practice-and-impact/rap_mvp_maturity_guidance/blob/master/Reproducible-Analytical-Pipelines-MVP.md). You can find a nicer, more readable version of the PEP 8 guidance [here](https://pep8.org/).
+
+Quick setup for working with Python
+===================================
+**To make this as easy as possible**
+
+Linting installation
+--------------------
+**What is Linting?**  
+A lint, or a linter, is a tool that analyzes your code to flag programming errors, bugs and stylistic errors. Simply put, a linter is a tool that scans your code with the goal of finding issues that could lead to bugs or inconsistencies with code health and style. 
+
+* You should have as a minimum [pylint](https://pylint.org/#install) installed in order to quality check code before merging any Pull Requests. You can find install instructions on the linked homepage and further setup instructions on their [github page](https://github.com/PyCQA/pylint). Here is a [nice tutorial](http://pylint.pycqa.org/en/latest/tutorial.html) should its usage not be clear.
```

Comment:
- Linting should be done in a Concourse pipeline. We should encourage local installation, but neither rely on it being installed locally or require it.
- Could we use [Black](https://black.readthedocs.io/en/stable/) over Pylint? Is there a specific need for Pylint?